static/src/javascripts/bootstraps/standalone.commercial.ts

```diff
@@ -104,7 +107,30 @@ const loadDcrBundle = async (): Promise<void> => {
 	return;
 };
 
-const loadModules = () => {
+const loadBaseModules = () => {
+	const modulePromises: Array<Promise<unknown>> = [];
+
+	commercialBaseModules.forEach((module) => {
+		const [moduleName, moduleInit] = module;
+
+		catchErrorsWithContext(
+			[
+				[
+					moduleName,
+					function pushAfterComplete(): void {
+						const result = moduleInit();
+						modulePromises.push(result);
+					},
+				],
+			],
+			tags,
+		);
+	});
+
+	return Promise.allSettled(modulePromises);
+};
```

Comment:
This looks very WET 😆.

I like the change of separating `base` and `extra` modules, but I think that we can then simply make the module array of tuples an argument to `loadModules`. What do you think?